app/src/features/toolbar/components/UserButton.tsx

```diff
@@ -37,35 +38,52 @@ function UserButton() {
       handleClose();
       navigate(route);
     },
-    [handleClose, navigate]
+    [handleClose, navigate],
   );
 
   const handleLogout = useCallback(() => {
     logout();
-    handleNavigate('/');
+    handleNavigate("/");
   }, [handleNavigate, logout]);
 
-  if (!!user) {
+  if (user) {
```

Comment:
Just checking if this should be `!!` to err on the side of caution, or it is safe by the time we get here?